src/libraries/System.IO.FileSystem/tests/Directory/Move.cs

```diff
@@ -33,6 +33,7 @@ public void EmptyPath()
         }
 
         [Fact]
+        [PlatformSpecific(~TestPlatforms.Browser)]
```

Comment:
Is this a bug, or does this really exercise a functionality not supported in browser?

Reply:
Still need to go through these and make sure they really are not supported or bugs or whatever.

Reply:
The VFS does have odd behavior for some of these tests.  We can likely work around some of that in the pal, but this suite in particular is going to need a bit of analysis like Ken was saying.

Reply:
We should have an issue tracking it then and use ActiveIssue instead. Thanks.

Reply:
What is the preferred way to comment on a platform specific test.  For instance the `public void FlushCanBeUsedOnPipes_Success(bool? flushToDisk)` test run into PNSE IO.Pipes.  Should this use an ActiveIssue?  Does not make sense to me for that type of case but want to make sure each case is documented for the PR.

Reply:
Is just a comment in the code sufficient for instance?  `[PlatformSpecific(~TestPlatforms.Browser)] // IO.Pipes not supported`

Reply:
If the problem is understood and by design, comment like what you have suggested looks good to me.